CHANGELOG.md

```diff
@@ -771,6 +775,10 @@ should use 4.0.1-alpha.0 for testing.
 
```

Comment:
Something like the following (ideally above `#### web3-utils` so titles are sorted alphabetically)
```suggestion

#### web3-plugin-example
- The package `web3-plugin-example` has been added with the basic functionality. (#5393)
```

Reply:
Updated via [this](https://github.com/web3/web3.js/pull/5393/commits/c939cbc428f00151ff392588b110d1cab12621a5) commit